rxsample/src/androidTest/java/com/mateuszkoslacz/moviper/rxsample/utils/RecyclerViewMatcher.java

```diff
@@ -0,0 +1,67 @@
+package com.mateuszkoslacz.moviper.rxsample.utils;
+
+import android.content.res.Resources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class RecyclerViewMatcher {
```

Comment:
The auto code reformat alters this class. Please perform reformat before commiting.